### PR TITLE
Editor: Add support for PLY point clouds.

### LINKED ISSUE #22133: Editor: Support for PLY point clouds
When this PLY file is dropped into the Editor, it is treated as a Mesh -- selecting `PointsMaterial` is not available as an option.

```code
ply
format ascii 1.0
element vertex 3
property float x
property float y
property float z
property uint8 red
property uint8 green
property uint8 blue
end_header
10.0 10.0 0.0 255 100 0
-10.0 10.0 0.0 0 100 255
0.0 0.0 0.0 100 255 0
```

`BufferGeometry` does not have a property to indicate what kind of geometry it represents.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: Fixed #22133

**Description**

The implementation of detecting point clouds is similar to how drc files are processed.

### CODE REVIEW

**Comment by Mugen87** [comment]
Can we avoid the linter warning `'XRWebGLLayer' is not defined ` by adding `XRWebGLLayer` to this section in `package.json`?

https://github.com/mrdoob/three.js/blob/c97a3e5598597b4f96365b1ff0323e04b8e92b8e/package.json#L41-L48

File: editor/js/Loader.js [hunk 1]
```diff
@@ -438,12 +438,27 @@ function Loader( editor ) {
...
+						var material = new THREE.MeshStandardMaterial();
+
+						object = new THREE.Mesh( geometry, material );
+						object.name = filename;
+
+					} else {
+
+						var material = new THREE.PointsMaterial( { size: 0.01 } );
+
+						if ( geometry.hasAttribute( 'color' ) === true ) material.vertexColors = true;
```
Comment (WestLangley) [inline editor/js/Loader.js:454]: maybe
```js
material.vertexColors = geometry.hasAttribute( 'color' );
```
as is done elsewhere in the file.

Comment (Mugen87) [reply editor/js/Loader.js:454]: Good catch!

**Comment by Mugen87** [comment]
> Can we avoid the linter warning 'XRWebGLLayer' is not defined by adding XRWebGLLayer to this section in package.json?

Let's give this a try.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
>> Can we avoid the linter warning 'XRWebGLLayer' is not defined by adding XRWebGLLayer to this section in package.json?
>
> Let's give this a try d7a756acfce186bd3ab4b7693829cbb49597be6e, ad8c600aa149bcfbcfbaf6566cfb0546df5942f3.

@marcofugaro Is that the right way of doing it?

**Comment by marcofugaro** [comment]
Yup, correct 👍

**Pull request merged**